lib/vscode/resources/web/code-web.js

```diff
@@ -274,26 +286,28 @@ const requestHandler = (req, res) => {
 	}
 };
 
-const server = http.createServer(requestHandler);
-server.listen(LOCAL_PORT, () => {
-	if (LOCAL_PORT !== PORT) {
-		console.log(`Operating location at         http://0.0.0.0:${LOCAL_PORT}`);
-	}
-	console.log(`Web UI available at           ${SCHEME}://${AUTHORITY}`);
-});
-server.on('error', err => {
-	console.error(`Error occurred in server:`);
-	console.error(err);
-});
+if (!args.server) {
```

Comment:
This adds partial support for running within VSCode’s `yarn web`, but it’s buggy and changes quite a bit in v1.59